.pylintrc

```diff
@@ -107,7 +107,8 @@ disable=raw-checker-failed,
         too-many-ancestors,
         too-many-nested-blocks,
         global-statement,
-        too-many-lines
+        too-many-lines,
+        line-too-long
```

Comment:
Disabling this might make formatting with black even more important. Furthermore, there'd be no visual indicator in the IDE that black will need to be run. What line(s) is this needed for?

Reply:
It seemed redundant since black already handles that, and black doesn't split long string literals.  But anyway I removed this change since as you noted I fixed it by splitting the string manually.